include/seqan3/search/detail/search_scheme_precomputed.hpp

```diff
@@ -81,64 +81,59 @@ using search_scheme_dyn_type = std::vector<search_dyn>;
  *          seems to be a good greedy approach.
  */
 template <uint8_t min_error, uint8_t max_error>
-inline constexpr int optimum_search_scheme{0};
+static constexpr int optimum_search_scheme{0};
 
 //!\cond
 
 template <>
-inline constexpr search_scheme_type<1, 1> optimum_search_scheme<0, 0>{{{{1}, {0}, {0}}}};
+constexpr search_scheme_type<1, 1> optimum_search_scheme<0, 0>{{{{1}, {0}, {0}}}};
```

Comment:
* `inline` breaks linkage
* `static` is not allowed for template specializations